cmd/genji/shell/shell.go

```diff
@@ -192,12 +202,13 @@ func (sh *Shell) executeInput(in string) error {
 		return sh.runCommand(in)
 	// If it ends with a ";" we can run a query
 	case strings.HasSuffix(in, ";"):
```

Comment:
I think we just need to ignore the query if it matches the condition we discussed in the issue.
Basically, I think all we need to do is this:
- trim the `in` variable to remove all withespaces
- ignore the command if `len(in) == 0`

Reply:
Indeed.